integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/car/deployment/test/CAppArtifactIndicationTestCase.java

```diff
@@ -39,38 +39,22 @@
 public class CAppArtifactIndicationTestCase extends ESBIntegrationTest {
     private ServiceAdminClient serviceAdminClient;
 
-    String service = "sampleCustomProxy";
-    String carFileName = "esb-artifacts-car_1.0.0.car";
+    String carFileName = "esb-artifacts-car";
 
-    @BeforeClass(alwaysRun = true)
-    protected void uploadCarFileTest() throws Exception {
+    @Test(groups = { "wso2.esb" }, description = "test car application deployment")
+    public void carDeploymentTest() throws Exception {
         super.init();
-        CarbonAppUploaderClient carbonAppUploaderClient = new CarbonAppUploaderClient(
-                context.getContextUrls().getBackEndUrl(), sessionCookie);
-        carbonAppUploaderClient.uploadCarbonAppArtifact("esb-artifacts-car_1.0.0.car", new DataHandler(
-                new FileDataSource(new File(getESBResourceLocation() + File.separator + "car" + File.separator
-                        + "esb-artifacts-car_1.0.0.car"))));
-        serviceAdminClient = new ServiceAdminClient(context.getContextUrls().getBackEndUrl(), sessionCookie);
+
+        Assert.assertTrue(checkCarbonAppExistence(carFileName),"car application deployment failed");
         TimeUnit.SECONDS.sleep(5);
```

Comment:
shall we remove these thread sleeps, since we have the capp in the server start itself , we wouldn't need it.

Reply:
Remove the thread sleeps. Please refer the latest commit.